fdbserver/storageserver.actor.cpp

```diff
@@ -747,6 +748,17 @@ bool SSPhysicalShard::hasRange(Reference<ShardInfo> shard) const {
 	return false;
 }
 
+struct TenantSSInfo {
+	constexpr static FileIdentifier file_identifier = 3253114;
+	TenantName name;
```

Comment:
I think we are now not using the name, so we could opt to remove it from this struct